### PR TITLE
Make player able to grab Big Bass bomb while fusing 

### LINKED ISSUE #280: Make player able to grab Big Bass bomb while fusing
### Description

Add the ability to the player to grab the bombs Big Bass throws while fusing, for which maybe the player can even throw the bomb back to the boss.

### Alternatives & Prior Art

Add a way to extend the bomb fusing animation, maybe ask an artist to make a longer animation.

### PR DESCRIPTION
**Pull request by Zac8668**
Closes #280

### CODE REVIEW

**Comment by Zac8668** [comment]
For now I just made the bomb animation fps longer, until we make a bigger animation to extend the time for the player to grab the bomb.

File: assets/items/bomb/bomb.item.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ kind: !Bomb
     columns: 11
     rows: 3
 
-    animation_fps: 0.12
+    animation_fps: 1.
```
Comment (odecay) [inline assets/items/bomb/bomb.item.yaml:14]:
```suggestion
    //TODO: return and find another way to extend duration after animation system changes
    animation_fps: 1.
```

File: src/fighter_state.rs [hunk 2]
```diff
@@ -1725,18 +1772,31 @@ fn grabbing(
...
+                                    consts::THROW_ITEM_OFFSET.y + image.image_size.y,
+                                    consts::PROJECTILE_Z,
+                                ));
+
                                 picked_item_ids.insert(item_ent);
                                 **fighter_inventory =
                                     Some(items_assets.get(item).expect("Item not loaded!").clone());
-                                commands.entity(item_ent).despawn_recursive();
+                                commands.entity(item_ent).remove::<Item>().insert(BeingHeld);
+                                commands.entity(fighter_ent).add_child(item_ent);
```
Comment (odecay) [inline src/fighter_state.rs:1799]: I like the new way of doing Holding by adding as child. :+1: 
Nice job simplifying.

**Review by odecay — CHANGES_REQUESTED** [review]
I think this is looking good, will test to make sure everything is working as expected.

**Review by odecay — APPROVED** [review]

**Comment by odecay** [comment]
Getting panic I *think* when the boss tries to throw a bomb.

```
thread 'Compute Task Pool (4)' panicked at 'Fighter has no item', src/fighter_state.rs:1372:18
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', /home/otis/.cargo/registry/src/github.com-1ecc6299db9ec823/bevy_tasks-0.9.1/src/task_pool.rs:273:45
```

Is there a YAML property that is missing for the boss in this PR?

**Review by odecay — APPROVED** [review]

**Comment by odecay** [comment]
bors merge

**Pull request merged**